src/SecurityManager.php

```diff
@@ -31,6 +31,9 @@
  */
 class SecurityManager
 {
+    public bool $preventGlobalNameSpacePollution = false;
+    public ?string $allowedNamespace = null;
```

Comment:
instead of public properties could we add a configuration object with getter/setter for those settings and then it is possible to pass the Configuration object to the SecurityManager at the Constructor? Inside the SecurityManager we then only need a single private property which holds the Configuration.